docs/SceneViewer.md

```diff
@@ -0,0 +1,134 @@
+# Scene Viewer
+
+The SceneViewer component allows you to render a specified [AWS IoT TwinMaker scene](https://docs.aws.amazon.com/iot-twinmaker/latest/guide/scenes.html) for viewing experience.
+
+## Setup
+
+The SceneViewer component renders assets including `.svg` and `.hdr` files. You will need to configure proper loader like [file-loader](https://v4.webpack.js.org/loaders/file-loader/) for your application.
+
+There is sample code in [examples/react-app](https://github.com/awslabs/iot-app-kit/tree/main/sample-app/examples/react-app) that shows how to use this component in detail.
+
+### Basic React component example
+
+```tsx
+import { initialize } from '@iot-app-kit/source-iottwinmaker';
+import { SceneViewer } from '@iot-app-kit/scene-composer';
+
+const sceneLoader = initialize({ ... }).s3SceneLoader('scene-id');
+
+<SceneViewer 
+  sceneLoader={sceneLoader}
+/>
+```
+
+## Properties
+
+The SceneViewer component contains the following properties that you can customize. 
+
+### `sceneLoader`
+
+The class to load scene metadata and content.
+
+**Note: When a new instance of this object is passed in, the SceneView will trigger a new loading of the whole scene. Therefore, do not recreate this object when not needed.**
+
+- Type: `SceneLoader` defined in `@iot-app-kit/source-iottwinmaker`
```

Comment:
remove list indentations for `Type` statements across docs, make consistent with existing documents

Reply:
addressed in https://github.com/awslabs/iot-app-kit/pull/256